github/compliance/issue.py

```diff
@@ -316,6 +316,27 @@ def labels_to_preserve():
         raise RuntimeError('this line should never be reached') # all cases should be handled before
 
 
+def close_issue_with_retry(
+    issue: github3.issues.ShortIssue,
+) -> bool:
+    '''
+    close issue with recursive retries
+    '''
+    try:
+        return issue.close()
+
+    except github3.exceptions.UnprocessableEntity:
+        # assignee was suspended from github
+
+        if not issue.assignees:
+            return False
```

Comment:
We do use the return value, see https://github.com/gardener/cc-utils/blob/0a0f8092736a53ada5019a00a2984fa876f25d03/github/compliance/issue.py#L354.
Agree on raise instead of `False`.